usrsctplib/netinet/sctp_timer.c

```diff
@@ -864,7 +864,7 @@ sctp_t3rxt_timer(struct sctp_inpcb *inp,
 
 				(void)SCTP_GETTIME_TIMEVAL(&now);
 				if (net->last_sent_time.tv_sec) {
-					ms_goneby = (now.tv_sec - net->last_sent_time.tv_sec) * 1000;
+					ms_goneby = (unsigned int)(now.tv_sec - net->last_sent_time.tv_sec) * 1000u;
```

Comment:
Fixed in https://github.com/sctplab/usrsctp/commit/8463f3adbc5b5bf7d5d9a02172750181fc213336.